__integrations__/utils/client.ts

```diff
@@ -49,3 +49,20 @@ export const fetchGenerateLcParams = (
       body: JSON.stringify(body)
     }
   );
+
+export const GET_ASSERTION_PATH = "api/v1/assertions";
+export const fetchGetAssertion = (
+  assertionRef: string,
+  jwtHeaderName: string,
+  jwt: string,
```

Comment:
to handle missing jwt this is optional
```suggestion
  jwt?: string,
```